extension/scripts/global/team.js

```diff
@@ -217,6 +217,13 @@ const TEAM = [
 		torn: 2383326,
 		color: "white", // No explicit color chosen.
 	},
+	{
+		name: "TravisTheTechie",
+		title: "Developer",
+		core: false,
+		torn: 3549588,
+		color: "white", // No explicit color chosen.

```

Comment:
You should choose a color. The one above you didn't put a team entry, and I didn't want to choose a color for them.